servicemanager/service/smplayservice.py

```diff
@@ -117,24 +119,38 @@ def start_from_binary(self):
                 print b.fail + "ERROR: could not start '" + self.service_name + "' " + b.endc
             return popen_output.pid
 
-    def _unzip_play_application(self):
+    def _unpack_play_application(self, extension):
```

Comment:
We kinda need that, as the Play apps are packaged with .zip in Nexus and .tgz in Bintray - so to pick the right file, we have to tell it what to look for.